scale/job/views.py

```diff
@@ -1004,7 +512,7 @@ def get_serializer_class(self):
             return JobTypeFailedStatusSerializerV6
         else:
             return JobTypeFailedStatusSerializer
```

Comment:
I added some v5 stuff here recently that needs to be cleaned up.

Reply:
Removed. Good catch!